vega-tooltip/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "vega-tooltip",
+  "author": "Zening Qu, Sira Horradarn",
+  "filename": "vega-tooltip.min.js",
+  "description": "A tooltip plugin for Vega-lite and Vega visualizations.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vega/vega-tooltip.git"
+  },
+  "license": "BSD-3-Clause",
+  "npmName": "vega-tooltip",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/vega-tooltip.+(js|css|map)"
```

Comment:
Same as in https://github.com/cdnjs/cdnjs/pull/10760#discussion_r103609110. We might add the compiled js files for every single ts file to build in the future.